mw-webapp/src/component/image/Image.tsx

```diff
@@ -25,21 +27,65 @@ interface ImageProps {
    * Data attribute for cypress testing
    */
   dataCy?: string;
+
+  /**
+   * Enable image zooming on click
+   */
+  zoomable?: boolean;
 }
 
 /**
- * Сomponent for displaying images
+ * Component for displaying images
  */
 export const Image = (props: ImageProps) => {
-  const className = clsx(styles.image, props.className);
+  const [isZoomed, setIsZoomed] = useState(false);
+
+  /**
+   * Handles the click event.
+   */
+  const handleClick = () => {
+    if (props.zoomable) {
+      setIsZoomed(!isZoomed);
+    }
+  };
+
+  const className = clsx(styles.image, props.className, props.zoomable && styles.zoomable);
+
+  /**
+   * Creates a portal for displaying a zoomed image overlay.
+   * If `isZoomed` is true, it renders an overlay with the zoomed image.
+   * Otherwise, it returns null.
+   */
+  const createZoomedPortal = () => {
```

Comment:
We already have modal component? so better to use it

@tritonJS826  What do you think?

Reply:
yeah, why not to use existent component? )